modules/installer/cfg/askconfig.go

```diff
@@ -0,0 +1,93 @@
+package cfg
+
+import (
+	"bufio"
+	"fmt"
+	logger "github.com/kuberlogic/kuberlogic/modules/installer/log"
+	"github.com/pkg/errors"
+	"io"
+	"strings"
+)
+
+func readString(reader *bufio.Reader, defaultValue string) (*string, error) {
+	line, err := reader.ReadString('\n')
+	if err != nil {
+		return nil, errors.New("could not read from reader")
+	}
+	line = strings.TrimSpace(line)
+	if line == "" {
+		return &defaultValue, nil
+	}
+
+	return &line, nil
+}
+
+func AskConfig(screen io.Reader, log logger.Logger, defaultCfgLocation string) *Config {
+	config := new(Config)
+
+	reader := bufio.NewReader(screen)
+	log.Infof("Config is not found, please answer several questions")
+
+	fmt.Println(fmt.Sprintf(`kubeconfig path: (default=%s)`, DefaultKubeconfigPath))
```

Comment:
I would suggest to change the prompt to a slightly different form:
```fmt.Printf(`kubeconfig path [%s]: `,  DefaultKubeconfigPath)```

In my opinion this will give a better understanding of what a user needs to do.

Reply:
It applies to all input fields